script/gen_crd/gen-kamelets-crd-api-config.json

```diff
@@ -13,7 +13,7 @@
             "docsURLTemplate": "https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.25/#{{lower .TypeIdentifier}}-{{arrIndex .PackageSegments -1}}-{{arrIndex .PackageSegments -2}}"
         },
         {
-          "typeMatchPrefix": "^github\\.com/apache/camel-k/pkg/apis/camel/v1\\.",
+          "typeMatchPrefix": "^github\\.com/apache/camel-k/v2/pkg/apis/camel/v1\\.",
```

Comment:
Is this related to the single module refactoring? or has this been another issue that is now fixed?

Reply:
Yes, when it was upgrade to v2.